bindings/pyroot/pythonizations/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ file(COPY ${ROOT_headers_dir}/ DESTINATION ${CMAKE_BINARY_DIR}/include/ROOT)
 
 foreach(val RANGE ${how_many_pythons})
   list(GET python_under_version_strings ${val} python_under_version_string)
+  list(GET python_major_version_strings ${val} python_major_version_string)
```

Comment:
Shouldn't we do all this here?
https://github.com/root-project/root/blob/master/bindings/pyroot/CMakeLists.txt

Since it is a meta-target for Pythonizations + Cppyy, that would be the place to put it, instead of inside of the CMakeLists of one of the inner targets of the meta-target.

We could have a loop on the Python versions in that `pyroot/CMakeLists.txt` and include the new elements you are adding here, which only concern the meta-target.

Reply:
I think it is not nice to put in CMakeLists/txt that basically has only ```add_subdirectories```...

Reply:
Well it has `add_subdirectories` to recursively go into the inner targets, but in my opinion that is the `CMakeLists` of the meta-target, that is the right level. Why would you configure the meta-target inside the inner target pythonizations and not the cppyy one?